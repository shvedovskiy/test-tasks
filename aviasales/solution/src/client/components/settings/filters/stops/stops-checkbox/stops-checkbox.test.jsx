/* eslint-disable no-undef */
import React from 'react';

import StopsCheckbox from './stops-checkbox';


describe('<StopsCheckbox />', () => {
  const props = {
    id: 'stops-0',
    checked: false,
    value: '0',
    onChange: jest.fn(),
  };
  const stopsCheckbox = shallow(<StopsCheckbox {...props} />);

  it('renders properly', () => {
    expect(stopsCheckbox).toMatchSnapshot();
  });
});
