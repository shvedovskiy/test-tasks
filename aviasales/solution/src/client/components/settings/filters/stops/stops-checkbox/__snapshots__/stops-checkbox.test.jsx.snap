// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StopsCheckbox /> renders properly 1`] = `
<styled__StopsCheckboxContainer>
  <styled__CheckboxTarget
    checked={false}
    id="stops-0"
    onChange={[MockFunction]}
    value="0"
  />
  <styled__CheckboxVisual />
</styled__StopsCheckboxContainer>
`;
